packages/core/lib/console.js

```diff
@@ -39,20 +39,19 @@ const processInput = (input, allowedCommands) => {
     }
 
     const normalizedCommand = cmd.toLowerCase();
-    if (validTruffleCommands.includes(normalizedCommand)) {
-      return allowedCommands.includes(normalizedCommand)
-        ? words.slice(1).join(" ")
-        : makeIIFE(`ℹ️ : '${cmd}' is not allowed within Truffle REPL`);
-    }
-    return makeIIFE(`ℹ️ : '${cmd}' is not a valid Truffle command`);
+    return validTruffleConsoleCommands.includes(normalizedCommand)
+      ? words.slice(1).join(" ")
+      : makeIIFE(
+          `ℹ️ : '${words[0]} ${cmd}' is not valid in Console environment.`
```

Comment:
You've combined these two error messages into one, but I feel like the second is worth keeping...

...except, does the second ever come up?  I don't think it does, right?  Because it'll get interpreted as JS and you'll get an error from that instead, right?

...OK, I guess maybe that error message can go after all!  This might be worth a comment though?

Reply:
Another good point. I'll think this one over and get back tomorrow.

Reply:
The latest changes distinguishes between excluded Truffle Commands, and invalid Truffle commands. Thanks for suggesting maintaining an exception list.